src/main/java/edu/gonzaga/Deck.java

```diff
@@ -1,40 +1,140 @@
 package edu.gonzaga;
 
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
 public class Deck {
+    // attributes
     private ArrayList<Card> drawPile = new ArrayList<Card>();
     private ArrayList<Card> discardPile = new ArrayList<Card>();
 
+    // methods
+    /**
+     * Shuffles the deck
+     * @param pile an array of Card objects
+     * @return void
+     * 
+     * @author Tony
+     */
     private void shufflePile(ArrayList<Card> pile) {
-
+        Collections.shuffle(pile);
     }
 
+    /**
+     * Constructor for the Deck class
+     * @param cities an array of City objects
+     * @return a Deck object
+     * 
+     * @author Tony
+     */
     public Deck() {
+        // add 48 basic cards
+        initializeBasicCards();
 
-    }
+        // add 5 event cards
+        initializeEventCards();
 
-    public void getDrawPile() {
+        // add 6 epidemic cards
+        for (int i = 0; i < 6; i++) {
+            drawPile.add(new EpdemicCard());
+        }
         
+        // shuffle the deck
+        shufflePile(drawPile);
     }
 
-    public void getDiscardPile() {
-
+    /**
+     * Initialize EventCards
+     * @return a Deck object
+     * 
+     * @Author Tony
+     */
+    private void initializeEventCards() {
+        // add 5 event cards
+        List <String> eventNames = List.of("Airlift", "Government Grant", "One Quiet Night", "Resilient Population", "Forecast");
+
+        for (String name : eventNames) {
+            drawPile.add(new EventCard(name));
+        }
     }
 
-    public void drawCard() {
+    private void initializeBasicCards() {
+        // Should somehow find a way to connect to the actual City class
+        // add 48 basic cards
+        
+    }
 
+    /**
+     * Returns the draw pile
+     * @return the draw pile
+     * 
+     * @author Tony
+     */
+    public ArrayList<Card> getDrawPile() {
+        return this.drawPile;
     }
 
-    public void drawBottomCard() {
+    /**
+     * Returns the discard pile
+     * @return the discard pile
+     * 
+     * @author Tony
+     */
+    public ArrayList<Card> getDiscardPile() {
+        return this.discardPile;
+    }
 
+    /**
+     * Discards a card
+     * @param card a Card object
+     * @return void
+     * 
+     * @author Tony
+     */
+    public void discardCard(Card card) {
+        discardPile.add(card);
     }
 
-    public void shuffle() {
+    /**
+     * Draws the first card from the deck
+     * @return the first card from the deck
+     * 
+     * @author Tony
+     */
+    public Card drawCard() {
+        // if the draw pile is empty, shuffle the discard pile and put it on top of the draw pile
+        if (drawPile.isEmpty()) {
```

Comment:
If the draw pile is empty in Pandemic, the game ends and shouldn't reshuffle the discard pile on top